templates/daemon.sysv.erb

```diff
@@ -1,32 +1,32 @@
 #!/bin/bash
 #
-#       /etc/rc.d/init.d/node_exporter
+#       /etc/rc.d/init.d/<%= @name %>
 #
-#       Daemonize the prometheus node exporter
+#       Daemonize the prometheus <%= @name %>
 #
 # chkconfig:   2345 95 20
-# description: Prometheus exporter for machine metrics, written in Go with pluggable metric collectors.
-# processname: node_exporter
-# pidfile: /var/run/node_exporter/pidfile
+# description: Prometheus Exporter <%= @name %>
+# processname: <%= @name %>
+# pidfile: /var/run/<%= @name %>/pidfile
```

Comment:
Should we change this to `/var/run/<%= @name %>/<%= @name %>.pid` for consistency with Upstart?